app/components/app_patient_page_component.html.erb

```diff
@@ -32,12 +39,16 @@
       <% end %>
     <% else %>
       <%= govuk_button_to(
-        "Assess Gillick competence",
-        session_patient_manage_consents_path(session, patient_id: patient.id, section: @section, tab: @tab),
-        secondary: true,
-        name: "consent", value: "self_consent",
-        prevent_double_click: true
-      ) %>
+            "Assess Gillick competence",
+            session_patient_manage_consents_path(
+              session, patient_id: patient.id,
+                       section: @section,
+                       tab: @tab,
```

Comment:
What's with this formatting? Haven't seen this style before.

Reply:
Rufo 🤷